pkg/infrastructure/gcp/clusterapi/operation.go

```diff
@@ -11,7 +11,7 @@ import (
 // WaitForOperationGlobal will attempt to wait for a operation to complete where the operational
 // resource is globally scoped.
 func WaitForOperationGlobal(ctx context.Context, projectID string, operation *compute.Operation) error {
-	ctx, cancel := context.WithTimeout(ctx, time.Minute*1)
+	ctx, cancel := context.WithTimeout(ctx, time.Minute*2)
```

Comment:
Do we even need to set a timeout here or should the caller set it? Right now the caller is setting a 3 minute timeout and we are adding a shorter timeout in this function. It seems like we should just have one or the other.